scripts/gm/README.md

```diff
@@ -0,0 +1,252 @@
+
+**Entries**:
+* `[global]` - the global section defines application-wide defaults. All the entries here can be overridden on a
+  per-node basis.
+* `gaiad_binary` - Path to the `gaiad` binary to use.
+* `ports_start_at`
+  * In the global section this defines the first free port to use for newly created nodes.
+    The value will be incremented (by 10) when a new node requires ports.
+  * In the per-node configuration this is the first port to allocate for the node.
+    A total of 10 ports will be allocated per node.
+* `home_dir`
+  * In the global section this defines the default folder where all node folders will be created.
+  * In the per-node configuration this is the folder for the node configuration.
+* `auto_maintain_config` - automatically update the `persistent_peers` and `unconditional_peer_ids` sections of the node
+  configuration.
+* `add_to_hermes` - this node should be part of the hermes config.
+* `network`
+  * Only in subsections: Define the network (the validator connection) for the full node.
+  * **Mandatory for full nodes**, does not exist for validator nodes.
+
+Note: the user is welcome to create additional nodes outside the scope of `gm` on the local machine but `gm` will only
+manage nodes that are added to the configuration file.
+
+### The network configuration
+**Where**: Default is the folder `$HOME/.gm/<node_name>`, but it can be configured in `gm.toml` using the `home_dir`
+entry.
+
+**Description**: The configuration and data folder for a node. Partially resembles a gaiad home folder (`.gaia`) but
+it has additional files to store the wallet mnemonics.
+
+**Entries**:
+* `config` - The node configuration folder. If the node is a full-node, the genesis file was copied from a validator
+config. The persistent_peers section is automatically managed if the node has the `auto_maintain_config` parameter
+  enabled in `gm.toml`.
+* `data` - The data folder.
+* `keyring-test` - the keyring folder as defined by `gaiad testnet` with the "test" keyring-backend.
+* `key_seed.json` - the node's signing and wallet key as defined using the `gaiad testnet` command.
+* `wallet_seed.json` - an extra wallet mnemonic defined on validator nodes with some tokens for developer use.
+* `pid` - the file that contains the process ID of the running node. (a la `/var/run`) Use `gm status` to see.
+* `log` - the log file that contains the output of the running node. Use `gm log <node>` to see.
+
+This setup allows developers to run a node outside of `gm` just by pointing the `gaiad --home-dir` to the folder.
+
+### Ports
+Ports are defined by the `ports_start_at` parameter which will be the first port assigned.
+Port assignment is as follows:
+```
+| name         | port redirection   |
+|==============|====================|
+| RPC (26657)  | ports_start_at + 0 | 
+| App (1317)   | ports_start_at + 1 | 
+| GRPC (9090   | ports_start_at + 2 | 
```

Comment:
The new gaia version includes a GRPC-Web which defaults to 9091

There's an enable so I can manually set that to false for now